src/Microsoft.AspNetCore.Authorization/DefaultAuthorizationService.cs

```diff
@@ -11,12 +11,21 @@
 
 namespace Microsoft.AspNetCore.Authorization
 {
+    /// <summary>
+    /// The default implementation of an <see cref="IAuthorizationService"/>.
+    /// </summary>
     public class DefaultAuthorizationService : IAuthorizationService
     {
         private readonly IAuthorizationPolicyProvider _policyProvider;
         private readonly IList<IAuthorizationHandler> _handlers;
         private readonly ILogger _logger;
 
+        /// <summary>
+        /// Creates a new instance of <see cref="DefaultAuthorizationService"/>.
+        /// </summary>
+        /// <param name="policyProvider">The <see cref="IAuthorizationPolicyProvider"/> used to provide policies.</param>
+        /// <param name="handlers">The handlers used to fufills <see cref="IAuthorizationRequirement"/>s.</param>
```

Comment:
This doc comment is very fufilling 😄 

`fufills` --> `fulfill` (add missing `l` and remove trailing `s`).